recipes/pecos/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pecos" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sandialabs/pecos/archive/{{ version }}.tar.gz
+  sha256: c9dcd2cc780937b0a1343d31b54d877b48e72907018d5bc96ba7f02455359e88
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
```

Comment:
In the `setup.py` there are version limits on several of the packages. Please match those here:

```
    'install_requires': ['numpy >= 1.10.4',
                         'pandas >= 0.18.0',
                         'matplotlib',
                         'jinja2'],
```

Reply:
Version limits have been added.